src/main/java/net/minecraftforge/server/command/CommandSetDimension.java

```diff
@@ -64,7 +64,7 @@ private static int execute(CommandSource sender, Collection<? extends Entity> en
         //    throw INVALID_DIMENSION.create(dim);
 
         entities.stream().filter(e -> e.dimension == dim).forEach(e -> sender.sendFeedback(new TranslationTextComponent("commands.forge.setdim.invalid.nochange", e.getDisplayName().getFormattedText(), dim), true));
-        entities.stream().filter(e -> e.dimension != dim).forEach(e ->  e.changeDimension(dim));
+        entities.stream().filter(e -> e.dimension != dim).forEach(e ->  e.changeDimension(dim, new PortallessTeleporter()));
```

Comment:
Shouldn't need to recreate this every invocation.  Or at the very least, just make it a lambda

Reply:
fixed.